services/datamanager/builtin/file_deletion.go

```diff
@@ -70,20 +70,23 @@ func exceedsDeletionThreshold(ctx context.Context, captureDirPath string, fsSize
 	}
 
 	err := filepath.WalkDir(captureDirPath, readSize)
-	if err != nil && !errors.Is(err, errAtSizeThreshold) {
+	if err != nil {
+		if errors.Is(err, errAtSizeThreshold) {
+			return true, nil
+		}
 		return false, err
 	}
-	if err == nil {
-		logger.Warnw("Not at threshold", "size", float64(dirSize)/fsSize, "threshold", captureDirToFSUsageRatio)
-
-	}
-	return err != nil && errors.Is(err, errAtSizeThreshold), nil
+	logger.Warnw("Not at threshold", "size", float64(dirSize)/fsSize, "threshold", captureDirToFSUsageRatio)
```

Comment:
[nit]: I think just remove before merge, otherwise will be pretty noisy for the case where file deletion is not necessary

Reply:
Yup! Sorry forgot to mark which are intended to stay vs go, most logging will be gone before I merge.